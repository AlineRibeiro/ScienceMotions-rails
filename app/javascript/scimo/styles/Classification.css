section#visual.classification {
  display: flex;
  flex-direction: column;
  flex-direction: column;
  height: 100%;
  margin-bottom: 40px;
}

section#visual.classification h1 {
  text-transform: uppercase;
  text-align: left;
}

div.phrase {
  font-family: "Space Mono", sans-serif;
  font-weight: 300;
  font-size: 1.2em;
  line-height: 1.75em;
  width: 85%;
}

.zero,
.two,
.six {
  color: #6ace10;
}

.one,
.four {
  color: #ed462f;
}

.three,
.five {
  color: #7f44ce;
}
