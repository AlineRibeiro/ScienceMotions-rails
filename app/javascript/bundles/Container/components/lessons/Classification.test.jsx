import React from 'react';
import renderer from 'react-test-renderer';
import Classification from './Classification';

describe('Classification', () => {
  it('renders correctly when there are no items', () => {
    const rendered = renderer.create(<Classification />).toJSON();
    expect(rendered).toMatchSnapshot();
  });
})
