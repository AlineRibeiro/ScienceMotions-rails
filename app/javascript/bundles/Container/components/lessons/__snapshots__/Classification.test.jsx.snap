// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Classification renders correctly when there are no items 1`] = `
Array [
  <section
    className="lesson-container"
    id="visual"
  >
    <div
      id="header"
    >
      <h1>
        Characteristics of living organisms 
      </h1>
      <input
        defaultValue="0"
        max="6"
        min="0"
        onChange={[Function]}
        type="range"
      />
    </div>
    <div
      className="phrase zero"
    >
      Movement - they can move and change their position.
    </div>
  </section>,
  <section
    id="text"
  >
    <div>
      <h1>
        Classification
      </h1>
      <p>
        The first letter of each of these characteristics makes up the acronym
        <span
          id="text-1"
        >
           MRS GREN
        </span>
        . This is a good way of remembering them.
      </p>
      <p>
        Living organisms can
        <span
          id="text-2"
        >
           reproduce
        </span>
      </p>
      <p>
        Living organisms can also control their
        <span
          id="text-3"
        >
           internal conditions
        </span>
        , such as their temperature or water content.
      </p>
    </div>
  </section>,
]
`;
